app-lifelike/src/main/java/com/example/lifelike/composable/LoggedOut.kt

```diff
@@ -39,8 +35,14 @@ interface LoggedOut {
 
                 when (val currentRouting = backStack.last()) {
                     Routing.Splash -> Splash.Content(onNext = currentRouting.next())
-                    Routing.RegUserName -> RegUserName.Content(user = user, onNext = currentRouting.next())
-                    Routing.RegUserPhone -> RegUserPhone.Content(user = user, onNext = currentRouting.next())
+                    Routing.RegUserName -> RegUserName.Content(
+                        user = user,
+                        onNext = currentRouting.next()
+                    )
+                    Routing.RegUserPhone -> RegUserPhone.Content(
+                        user = user,
+                        onNext = currentRouting.next()
+                    )
```

Comment:
I'd like to keep code formatting in most cases as it was before. I agree with some of the code style modifications, but not all. The PR should ideally introduce changes only in its scope (migrating to dev08).